apps/sms/views/conversation/js/conversation.js

```diff
@@ -2277,10 +2165,7 @@ var ConversationView = {
     this.enableConvertNoticeBanners();
 
     // If there was a draft, it just got sent so delete it.
-    if (this.draft) {
-      Drafts.delete(this.draft).store();
-      this.draft = null;
-    }
+    this.updateDraft();
```

Comment:
Good, so I'll leave it here so that we move it altogether in the  https://bugzilla.mozilla.org/show_bug.cgi?id=1136147